source/system/Demo_User_Guides/TI_Apps_Launcher_User_Guide.rst

```diff
@@ -365,14 +365,70 @@ _________________________
 
 3. Once you enter the necessary values for both HTTPS and No proxy, click on `Set Proxy` to set the proxy configuration.
 
+
+Using terminal
+______________
+
+.. ifconfig:: CONFIG_sdk in ('SITARA')
+
+    The terminal button on the left panel can be used to open a terminal application, which is
+    based on QMLtermwidget.
+
+    .. Image:: /images/terminal.png
```

Comment:
For all Image references,

Checkout https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace for indentations of height attribute with Image directive